py-jupyter.md

```diff
@@ -0,0 +1,249 @@
+since the code for creating the figures is kept together with the figures in the notebook.
+Notebooks can be exported to HTML, PDF, and many other formats,
+which facilitates sharing your results with others,
+and publishing online which we will learn more about in the publishing charter.
+<!--com: TODO link to pub -->
+Exporting can be done via `File --> Export Notebook As...`.
+When you export to PDF for the first time,
+you might be asked to download the Latex packages,
+which are needed to create the PDF.
+
+JupyterLab can run other applications than notebooks,
+e.g. there is a Python console, a text editor, and a terminal emulator.
+These can be opened via the launcher page or `File --> New`.
+Application can be placed side by side through drag and drop,
+and all currently running applications can be viewed
+in the "Running" tab in the sidebar.
+The next icon in the sidebar is the "Commands Palette" (the color palette icon),
+which can be used to search among all available toolbar commands for the notebook,
+e.g. you can search for "Run All" to find that command that runs all cells
+(also available via `Run --> Run all cells` in the toolbar menu).
+Conveniently,
+the command palette also shows a command's keyboard shortcut.
+The last tab (the wrench icon) is for advanced cell settings,
+which is outside the scope of this material.
+
+### How to get help {#py-how-to-get-help}
+
+ <!--TODO Depending on where this section will be in the book, some of these concepts might not have been introduced (packages, lists). -->
+
+Once you start out using Python,
+you don't know what functions are available within each package.
+If you're working with the `numpy` package in a Jupyter notebook,
+you can type `numpy.`<kbd>Tab</kbd> (that is numpy + period + tab-key)
+and a small menu will pop up showing all the available functions in the `numpy` module.
+There are plenty of available functions,
+so it can be helpful to filter the menu
+by typing the initial letters of the function name.
+
+ <!--TODO Add screenshots? -->
+To get more info on the function you want to use,
+you can type out the full name and then press <kbd>Shift + Tab</kbd>
+to bring up a help dialogue.
+For the `numpy.mean` function
+you can see that the first argument `a` should be "array-like".
+An array is essentially just a sequence of numbers,
+such as a list `[]` or tuple `()`.
+Instead of manually activating the pop up menu every time,
+JupyterLab offers a tool called "Contextual Help",
+which displays help information automatically as you type.
+This very useful and it is a good habit to always have it open next to the Notebook.
+More help is available via the "Help" menu,
+which links to useful online resources
+(for example `Help --> JupyterLab Reference`).
+The unformatted docstring can be seen by prepending or appending a `?` to the function name, e.g. `numpy.mean?`.
+Two `?` (`numpy.mean??`) will show the full source code for the function.
```

Comment:
Move these sentences above "More help is available..." since they're more about functions and the "Help" menu is more general.